app/routes/subsidy/measure-offer.js

```diff
@@ -0,0 +1,22 @@
+import Route from '@ember/routing/route';
+import DataTableRouteMixin from 'ember-data-table/mixins/route';
+import { inject as service } from '@ember/service';
+
+export default class SubsidyMeasureOfferRoute extends Route.extend(DataTableRouteMixin) {
+  @service currentSession;
+  @service store;
+
+  async model({ id: measureOfferID }) {
+    let groupId = this.currentSession.group.id;
+    return await this.store.findRecord('subsidy-measure-offer', measureOfferID, {
+      include: 'series',
+      filter: {
+        participations: {
+          'participating-bestuurseenheid': {
+            id: groupId,
+          },
+        },
+      },
+    });
```

Comment:
```suggestion
  async model({ id: measureOfferID }) {
    return await this.store.findRecord('subsidy-measure-offer', measureOfferID, {
      include: 'series',
    });
```
Not needed I believe, same results when I run it without